test/runtests.jl

```diff
@@ -96,23 +96,31 @@ end
     y = -2x + 10
     z = -3x
     @test @inferred(Measurements.cov([x, y, z])) ≈ [0.01 -0.02 -0.03; -0.02 0.04 0.06; -0.03 0.06 0.09]
+    @test [@inferred(Measurements.cov(x,x)), @inferred(Measurements.cov(y,y)), @inferred(Measurements.cov(z,z))] ≈ [0.01, 0.04, 0.09]
```

Comment:
How about comparing also to the elements of `Measurements.cov([x, y, z])` for consistency?  E.g. something like
```suggestion
    @test [@inferred(Measurements.cov(x,x)), @inferred(Measurements.cov(y,y)), @inferred(Measurements.cov(z,z))] ≈ [0.01, 0.04, 0.09]
    @test [@inferred(Measurements.cov(x,x)), @inferred(Measurements.cov(y,y)), @inferred(Measurements.cov(z,z))] ≈ diag(Measurements.cov([x, y, z]))
```
Similarly below.